src/svg/moon.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
Let's the change the directory name from `svg` to `resources` and then create another directory inside called `images` and have this live there.